app/models/user.rb

```diff
@@ -29,6 +29,8 @@
 class User < ActiveRecord::Base
   include DeviseTokenAuth::Concerns::User
 
+  after_update :handle_subscription
+
```

Comment:
Ah having trouble testing this callback here;

Not sure why it's not being fired in my specs. Think i'm doing something dumb but lmk if you catch something quick!